DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -136,8 +136,8 @@ void L1TStage2uGT::analyze(const edm::Event& evt, const edm::EventSetup& evtSetu
    // Get uGT algo bit statistics
    else {
       //algoBits_->Fill(-1.); // fill underflow to normalize // FIXME: needed? 
-      for (int ibx=uGtAlgs->getFirstBX(); ibx <= uGtAlgs->getLastBX(); ++ibx) {
-         for (auto itr = uGtAlgs->begin(ibx); itr != uGtAlgs->end(ibx); ++itr) { // FIXME: redundant loop over 1-dim vector?
+      for (int ibx = uGtAlgs->getFirstBX(); ibx <= uGtAlgs->getLastBX(); ++ibx) {
```

Comment:
You have inserted a space here, but it is missing from 81x.
Since this PR is already uses as a base for few others, I will ask for it to be merged.
But you should update the 81x version of this to include these spaces, even if they don't really change anything :)